From 13f219622714ed2beba238fc74e600d43e3ac127 Mon Sep 17 00:00:00 2001
From: Wyon Bi <bivvy.bi@rock-chips.com>
Date: Sun, 30 Sep 2018 15:30:32 +0800
Subject: [PATCH] phy: add MIPI/LVDS/TTL modes to the phy_mode enum

This patch adds more generic PHY modes to the phy_mode enum, to
allow configuring generic PHYs to the MIPI/LVDS/TTL mode by
using the set_mode callback.

Change-Id: Ib6966828011aa52f1f133449f69df46c2001a57b
Signed-off-by: Wyon Bi <bivvy.bi@rock-chips.com>
---
 include/linux/phy/phy.h | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/include/linux/phy/phy.h b/include/linux/phy/phy.h
index 43b2da6f4be2..9787843f0578 100644
--- a/include/linux/phy/phy.h
+++ b/include/linux/phy/phy.h
@@ -29,6 +29,9 @@ enum phy_mode {
 	PHY_MODE_USB_HOST,
 	PHY_MODE_USB_DEVICE,
 	PHY_MODE_USB_OTG,
+	PHY_MODE_VIDEO_MIPI,
+	PHY_MODE_VIDEO_LVDS,
+	PHY_MODE_VIDEO_TTL,
 };
 
 /**
-- 
2.35.3

